numpy/add_newdocs.py

```diff
@@ -5463,18 +5442,22 @@ def luf(lamdaexpr, *args, **kwargs):
 
     Parameters
     ----------
-    X : array_like
-        First input array.
-    Y : array_like
-        Second input array.
-    out : array_like
-        An array to store the output. Must be the same shape as the
-        output would have.
+    *x : array_like
```

Comment:
Well, this doesn't even get rendered, so no danger of it getting rendered incorrectly!

Since the only place this is ever seen is in the terminal, I'd argue it's better without backslashes.

I'd also argue that to keep the terminal tidy, the fix should go in the numpy sphinx extension, not here